CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ if(NOT CMAKE_BUILD_TYPE)
         "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
+option(COMPILE_PYTHON3_BINDINGS "Enable/Disable Python bindings. Python bindings are disabled by default." OFF)
```

Comment:
This option is already defined on line 179 when not building on Android:
https://github.com/prophesee-ai/openeb/blob/main/CMakeLists.txt#L179

Also, is there a reason you set it to off by default? If building python bindings on MacOS is problematic, it can be disabled by default for this platform.

Reply:
> *This option is already defined on line 179 when not building on Android*

Hey, yeah I didn't catch that definition on line 179.

> *If building python bindings on MacOS is problematic, it can be disabled by default for this platform.*

I am able to build the python bindings on MacOS.

> *Also, is there a reason you set it to off by default?*

The main reason I thought to change this behavior is to present the simplest build case first. As I walked through the documentation trying to understand what was needed and what wasn't needed to build and use the C++ library, I realized gtest, pytest and pybind11 were all optional.
I think by giving users the option to ENABLE features like testing and python bindings empowers them to:
- learn more about the product features incrementally
- avoid having to install/remove packages that the don't actually need,

I added [new documentation](https://github.com/emsight/openeb/blob/feature/openeb-updates/docs/build-and-installation-guides/macos-build-test-and-installation.md#additional-build-configurations). I initially wrote it keeping the options ON be default. I personally found the documentation and build experience better with the options OFF by default so I went with it... but that's just my take.

Reply:
@lbristiel-psee @ogeorget-psee what do you think?